src/mmg3d/API_functionsf_3d.c

```diff
@@ -137,6 +137,24 @@ FORTRAN_NAME(MMG3D_SET_OUTPUTSOLNAME,mmg3d_set_outputsolname,
   return;
 }
 
+/**
+ * See \ref MMG3D_Set_inputParamName function in \ref common/libmmgcommon_private.h file.
+ */
+FORTRAN_NAME(MMG3D_SET_INPUTPARAMNAME, mmg3d_set_inputparamname,
+             (MMG5_pMesh *mesh,char* fparamin, int* strlen0, int* retval),
+             (mesh,fparamin,strlen0,retval)) {
+
+  char *tmp = NULL;
+
+  MMG5_SAFE_MALLOC(tmp,*strlen0+1,char,return);
+  strncpy(tmp,fparamin,*strlen0);
+  tmp[*strlen0] = '\0';
+  *retval = MMG3D_Set_inputParamName(*mesh,tmp);
```

Comment:
No need to change this but MMG5_Set_inputParaName can be called directly (it has the same result but if someone searches what is doing this functions, it allows to access the code source faster (you have one less redirection).
If the function was called a huge amout of time (not the case here), it is also possible that adding one useless function call will slightly slow down the code.

Reply:
Noted for future PR. Thanks.